src/pybind/mgr/volumes/fs/operations/group.py

```diff
@@ -104,7 +104,11 @@ def create_group(fs, vol_spec, groupname, pool, mode, uid, gid):
     """
     group = Group(fs, vol_spec, groupname)
     path = group.path
-    fs.mkdirs(path, mode)
+    vol_spec_base_dir = group.vol_spec.base_dir.encode('utf-8')
+
+    # create vol_spec base directory with default mode(0o755) if it doesn't exist
+    mkdir0(fs, vol_spec_base_dir, vol_spec.DEFAULT_MODE)
```

Comment:
I'm confused by `mkdir0`. Why wouldn't you just do `mkdirs`?

Reply:
I have rename `mkdir0` to `create_base_dir`.